src/components/Navigation/NavigationItem.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<li class="inline-block font-bold uppercase text-md hover:text-accent">
+	<li class="block mr-auto sm:mr-8 sm:inline-block font-bold uppercase text-md hover:text-accent">
```

Comment:
A little y-padding would come handy here. Looks too tight.

Also, could we add shadow to the whole wrapper and lighter primary to hover?

![image](https://user-images.githubusercontent.com/27869979/98372947-d0519200-203e-11eb-88ea-9b58410d2e3c.png)


![image](https://user-images.githubusercontent.com/27869979/98372795-9da79980-203e-11eb-8061-9d7d125351ce.png)